playpen/vagrant-setup.sh

```diff
@@ -64,10 +64,7 @@ fi
 
 # install rpms, then remove pulp*
 echo "installing RPMs"
-sudo dnf install -y @pulp-server-qpid @pulp-admin @pulp-consumer
-sudo dnf remove --setopt=clean_requirements_on_remove=false -y pulp-\* python-pulp\*
-sudo dnf install -y git mongodb mongodb-server python-debian python-django python-glanceclient \
-                    python-keystoneclient python-mongoengine python-paste python-qpid-qmf \
+sudo dnf install -y git mongodb mongodb-server python-gofer-qpid python-qpid python-qpid-qmf \
```

Comment:
Does this need to include `qpid-tools` or is that part of a spec? Or do we not need that?